charts/jenkins/templates/jenkins-controller-statefulset.yaml

```diff
@@ -301,6 +307,11 @@ spec:
       - name: plugin-dir
         emptyDir: {}
       {{- end }}
+      {{- if .Values.controller.additionalSecrets }}
```

Comment:
you can already put whatever you want with https://github.com/nxtlytics/jenkins-helm-charts/blob/jenkins-secrets/charts/jenkins/templates/jenkins-controller-statefulset.yaml#L205-L207 ?

Reply:
Correct, this would just simplify the process of creating a kubernetes secret and mounting it in the place jenkins expects it